service/src/main/java/uk/gov/hmcts/reform/fpl/service/document/ManageDocumentService.java

```diff
@@ -278,6 +281,58 @@
         return setDateTimeUploadedOnSupportingEvidence(currentSupportingDocuments, previousSupportingDocuments);
     }
 
+    public List<Element<SupportingEvidenceBundle>> getRespondentStatementFurtherEvidenceCollection(CaseData caseData,
+                                                                                                   UUID id) {
+        Optional<Element<RespondentStatement>> respondentStatement = caseData.getRespondentStatementByRespondentId(id);
```

Comment:
can be written as:
       return caseData.getRespondentStatementByRespondentId(id)
            .map(Element::getValue)
            .map(RespondentStatement::getSupportingEvidenceBundle)
            .orElse(defaultSupportingEvidences());